packages/ui-kit/src/components/Select/Select.tsx

```diff
@@ -54,8 +54,8 @@ const SelectComponent = <TData extends string = string>(
             {placeholder}
           </option>
         )}
-        {options.map(({ label, value, disabled = false }) => (
-          <option value={value} disabled={disabled} key={value}>
+        {options.map(({ label, value, disabled = false }, index) => (
+          <option value={value} disabled={disabled} key={index}>
```

Comment:
This should also do the trick:

```suggestion
        {options.map(({ label, value, disabled = false }) => (
          <option value={value} disabled={disabled} key={label + '_' + value}>
```

Reply:
It definitely work for the case, but IMO it’s more like a hack instead of a common component implementation. Anyway the trick is also acceptable for me.